### ISSUE TITLE
python.django.correctness.model-save.django-db-model-save-super False Positive

### ISSUE THREAD

**Issue by tmiller02** [returntocorp/sgrep-rules#1106]
**Describe the bug**

The `python.django.correctness.model-save.django-db-model-save-super` rule appears to raise a false positive when the zero argument form of `super` is used to call the `.save` method.

**To Reproduce**

Create a python file containing the following:

```
from django.db import models

class MyModel(models.Model):

    def save(self, *args, **kwargs):
        print("my overriden save method")
        super().save(*args, **kwargs)


class MyOtherModel(models.Model):

    def save(self, *args, **kwargs):
        print("my overriden save method")
        super(MyOtherModel, self).save(*args, **kwargs)
```

and run semgrep:

```
semgrep --config p/r2c-best-practices mymodel.py
```

The first class is flagged by the `python.django.correctness.model-save.django-db-model-save-super` rule, and the second is not. 

**Expected behavior**

Both forms of calling the super `.save` method are equivalent, so neither should be flagged by this rule - https://docs.python.org/3/library/functions.html#super
